projects/packages/videopress/src/client/admin/components/input/index.tsx

```diff
@@ -163,6 +158,19 @@ export const SearchInput = ( {
 			type="text"
 			onEnter={ onEnterHandler }
 			onChange={ onChangeHandler }
+			endAdornment={
+				<>
+					{ componentProps.value !== '' && (
```

Comment:
Are we going to check for `undefined` here or should we open another issue?

Reply:
@dhasilva good point, I will map for empty, since we force `type='text'` for this component

Reply:
done https://github.com/Automattic/jetpack/pull/26206/commits/4e3a3b09bdcb46c175e75b6f162a147195ff6337